src/assets/icons/icon-trash-dark.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
+  <g fill="#32363f" fill-rule="evenodd" transform="translate(-96 -32)">
```

Comment:
Yes, this is what I used to do as well and it worked great. But as you said, it only works for inline SVG so I was only using that, whereas the current approach in cozy is to use background everywhere. Unless there's a reason to prefer backgrounds, I'd be in favor of switching to inline svg tbh.